File: problemas/1000/1036.cs
using System;

namespace uri1036
{
    internal static class Program
    {
        private static void Main()
        {
            string[] entrada = Console.ReadLine().Split(' ');
            double a = double.Parse(entrada[0]);
            double b = double.Parse(entrada[1]);
            double c = double.Parse(entrada[2]);
            double delta = b * b - 4.0 * a * c;

            if (delta < 0 || a == 0)
            {
                Console.WriteLine("Impossivel calcular");
            }
            else
            {
                double r1 = (-b + Math.Sqrt(delta)) / (2.0 * a);
                double r2 = (-b - Math.Sqrt(delta)) / (2.0 * a);
                Console.WriteLine($"R1 = {r1:F5}");
                Console.WriteLine($"R2 = {r2:F5}");
            }
        }
    }
}

File: problemas/1000/1036.py
import math

entrada = input().split(' ')
a = float(entrada[0])
b = float(entrada[1])
c = float(entrada[2])
delta = b * b - 4 * a * c

if delta < 0 or a == 0:
    print('Impossivel calcular')
else:
    r1 = (-b + math.sqrt(delta)) / 2 * a
    r2 = (-b - math.sqrt(delta)) / 2 * a
    print('R1 = {:.5f}'.format(r1))
    print('R2 = {:.5f}'.format(r2))
